tests/attach_readmem.rs

```diff
@@ -1,5 +1,6 @@
 //! This is a simple test to attach to already running debugee process
 
+#[cfg(unix)]
```

Comment:
Please use `#![cfg(unix)]`. This test uses `fork` which is only available on unix. While the actual test is currently `#[cfg(target_os = "linux)]`, the goal is that it will run on all unix systems.

Reply:
yeah, i was bit confused because the build wasn't failing for linux or bsd